app/src/main/kotlin/com/exallium/todoapp/repository/StubNoteMapperImpl.kt

```diff
@@ -12,7 +12,10 @@ class StubNoteMapperImpl(private val idFactory: IdFactory) : DataMapper<Note>, Q
     init {
         db.putAll((1..5).map {
             val id = idFactory.createId()
-            Pair(id, Note(id, "title$it", "body$it", 0, 0))
+            // set dates for some entries only in order to see UI with and without dates
+            val created : Long = if (it % 2 == 0) System.currentTimeMillis() else 0
```

Comment:
We probably want to utilize JodaTime.

At the very least, utilize a Proxy for System.currentTimeMillis()

Like:

```
class TimeProxy {
   fun getCurrentTimeMillis() = System.currentTimeMillis()
}
```

This allows you to mock out that portion (and prevents bringing system dependency into our Repository implementation)